src/app/core/services/constants.js

```diff
@@ -13,4 +13,10 @@ export default function (app) {
     app.constant('weatherAPIconstants', {
         'APIkey': 'b0f2a7928e6c418baf01f03c9101c5df'
     });
+    app.constant('homeConstants', {
+        'headers':  ['Amount From', 'Amount To', 'Commission', 'Currency From', 'Currency To', 'Date', 'Rate']
```

Comment:
this is not a global constants, put it just into controller

Reply:
have done it